src/Management/src/Endpoint/Actuators/Health/HealthEndpointMiddleware.cs

```diff
@@ -52,20 +52,19 @@ private string GetRequestedHealthGroup(PathString requestPath)
     {
         string[] requestComponents = requestPath.Value?.Split('/') ?? [];
 
-        if (requestComponents.Length > 0)
+        if (requestComponents.Length > 0 && requestComponents[^1] != _endpointOptionsMonitor.CurrentValue.Id)
         {
             return requestComponents[^1];
         }
 
-        _logger.LogWarning("Failed to find anything in the request from which to parse health group name.");
+        _logger.LogTrace("Did not find a health group in the request path.");
```

Comment:
Wouldn't it be more useful to log if we _do_ find a group, and what the group's name is?

Reply:
I was trying to preserve what I thought to be the original intent of logging when a group could/should have been found but wasn't... trace-level logging for both cases is reasonable and easy